cpp/src/arrow/compute/kernels/scalar_nested_test.cc

```diff
@@ -225,6 +225,56 @@ TEST(TestScalarNested, StructField) {
   }
 }
 
+TEST(TestScalarNested, MapArrayLookupNonRecursive) {
```

Comment:
This test passes.

Reply:
Yes of course, just wanted to check that at least the basic functionality worked.